frontend/src/api/Product.js

```diff
@@ -1,5 +1,7 @@
 import axios from "axios";
 import {BASE_URL, PAGE_SIZE} from "./Commons";
+import { storage } from "config/firebase";
+import { ref, uploadBytes, getDownloadURL } from "firebase/storage";
```

Comment:
Wait, if we're using firebase, why do we have all of the backend logic for uploading files to a folder on backend?